### ISSUE TITLE
MC2 DCA: Unable to validate PersonView manifest

### ISSUE THREAD

**Issue by vpchung** [Sage-Bionetworks/data_curator#369]
**Describe the bug**
Currently experiencing issues with validating a PersonView manifest.  To help with debugging, I reduced the manifest to a single row and received the following logs:

```
2022-06-17T04:48:20.283805+00:00 shinyapps[5992584]: ERROR: [2022-06-17 04:48:19] root - For attribute Person Grant Number in row 2 it does not appear as if you provided a comma delimited string. Please check your entry ('Not Associated'') and try again.
2022-06-17T04:48:20.283870+00:00 shinyapps[5992584]: ERROR: [2022-06-17 04:48:19] root - For attribute Person Consortium Name in row 2 it does not appear as if you provided a comma delimited string. Please check your entry ('PS-ON'') and try again.
2022-06-17T04:48:20.283926+00:00 shinyapps[5992584]: ERROR: [2022-06-17 04:48:20] root - For attribute Working Group Participation in row 2 it does not appear as if you provided a comma delimited string. Please check your entry ('Cell and Tissue Mechanics'') and try again.
2022-06-17T04:48:20.283975+00:00 shinyapps[5992584]: ERROR: [2022-06-17 04:48:20] root - For attribute Chair Roles in row 2 it does not appear as if you provided a comma delimited string. Please check your entry ('None'') and try again.
2022-06-17T04:48:20.284022+00:00 shinyapps[5992584]: Warning: Error in bind_rows: Can't combine `..1$Column` <character> and `..5$Column` <integer>.
2022-06-17T04:48:20.284086+00:00 shinyapps[5992584]:   [No stack trace available]
```

@brynnz22 was able to successfully validate the manifest using schematic CLI; ergo, the conclusion is that the error is with the DCA.

**To Reproduce**
1. Go to [the DCA](https://sagebio.shinyapps.io/csbc_data_curator_staging/) > **Submit & Validate Metadata**
2. In the **Selection** bar up top, select "CCKP- MC2 Admin" for Project, "people" for Folder, and "Person View" for template. Apply the changes by clicking on :arrows_counterclockwise:
3. Upload the [PersonView manifest](https://docs.google.com/spreadsheets/d/1_havAEvHe3FYjXaPLGkxpSqYLIR6PAYztn_0B0z5NDA/edit#gid=0) as a CSV > **Validate Metadata**
4. See error (hidden under the Waiter window)

https://user-images.githubusercontent.com/9377970/174228466-90f71e52-a2c6-42d8-a3c6-42ddd2786833.mov

**Priority** (select one)
- [ ] Minor ⬇️
- [x] Major 📢
- [ ] Critical 🆘

**Comment by vpchung** [comment]
Tagging @milen-sage @ychae as requested.  Thank you!

**Comment by ychae** [comment]
@vpchung is this part of the July 31 CSBC deadline? or does this flow into MC2?

**Comment by vpchung** [comment]
@aclayton555 may have the better answer, but I think in time for MC2?  We are okay with using the CLI for now, but we'd like our investigators to use the DCA.

**Comment by aclayton555** [comment]
@ychae  Re: deadline, we actually now have a no cost extension for CSBC/PS-ON through September to bridge us into MC2 (this is very recent, and I don't think this was in place before our last stakeholder meeting). That being said, we should aim to have this fixed by the end of August 2022, as the DCA is expected to be operational (i.e. tested and ready for external use) with the launch of MC2 (beginning of Sept 2022)

**Comment by rrchai** [comment]
@vpchung It is a familiar bug I recently encountered as well. It was due to the inconsistent value types from validation results and the `bind_rows()` will complain about that. A quick fix I think is to add this snippet to make all validation values to string type:

https://github.com/Sage-Bionetworks/data_curator/blob/6bc60d2465b81ec4eb426510ef466bb394fda05e/functions/validationResult.R#L81-L82

Feel free to try if that fixes the issue. I could make a hotfix to address this issue and update `main` branch of this repo tomorrow .
